packages/desktop-client/src/components/accounts/Reconcile.tsx

```diff
@@ -164,6 +167,11 @@ export function ReconcileMenu({
           />
         </InitialFocus>
       )}
+      <Text style={{ color: theme.pageTextSubdued, paddingBottom: 4 }}>
```

Comment:
Wondering if adding a little more padding would be nice? Right now it looks a little cramped to me, but maybe it's just me

Reply:
Bumped up the padding a little

<img width="280" alt="image" src="https://github.com/user-attachments/assets/b8ea523a-5133-47d9-9cfb-e8c9b739628d" />